### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mazmol1**
This has been cleaned up and re written

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+Cydia impactor:http://www.cydiaimpactor.com/
+
+
+ipa https://yalu.qwertyoruiop.com/yalu102_alpha.ipa (Current version at current date 26-01-17)
+
+
+sha1 2fe14f1c1e1a0d26203bbb123f6747a978dd2b4f
+
+
+mobilesubstrate works on devices that have sucesfully have cydia installed
```
Comment (BarakaAka1Only) [inline README.md]: *successfully

Comment (SuBLiNeR) [inline README.md]: *that have been successfully installed cydia

Comment (mwoolweaver) [inline README.md]: *that have successfully installed Cydia

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
-# yalu102
-incomplete iOS 10.2 jailbreak for 64 bit devices by qwertyoruiopz and marcograssi
+
+# yalu102 for **6s/SE and iPad Pro 10.0/10.1/10.2**
+
+**THE ISSUE TAB IS NOT A FORUM THREAD PLEASE DONT POST OR BEG FOR HELP EITHER GO TO REDDIT**
```
Comment (ryanb93) [inline README.md]: either go to reddit or what? 

Needs a comma after EITHER

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+
+
+Offsets may have to be added, if you know your offset for your device, and its the one listed **6s/SE and iPad Pro 10.0/10.1/10.2**
+
+
+Please do a Pull request with offset included.
```
Comment (ryanb93) [inline README.md]: 'Pull Request'

**Comment by jonisc** [comment]
"You might have to add offsets. If you know the offset of your device, and your device is listed (e.g. 6s/SE and iPad Pro  10.0/10.1/10.2),

Please submit a Pull request with the offset included"
My suggestion 😀

**Comment by Mila432** [comment]
who needs offsets , all devices are already added

**Comment by mazmol1** [comment]
Okay i give up :D

**Pull request closed without merging**